src/main/java/net/mcreator/ui/modgui/ItemGUI.java

```diff
@@ -474,11 +434,11 @@ else if (page == 0)
 		guiBoundTo.setSelectedItem(item.guiBoundTo);
 		inventorySize.setValue(item.inventorySize);
 		inventoryStackSize.setValue(item.inventoryStackSize);
-		hasDispenseBehavior.setSelected(item.hasDispenseBehavior);
-		dispenseSuccessCondition.setSelectedProcedure(item.dispenseSuccessCondition);
-		dispenseResultItemstack.setSelectedProcedure(item.dispenseResultItemstack);
-
-		updateDispenseElements();
+//		hasDispenseBehavior.setSelected(item.hasDispenseBehavior);
+//		dispenseSuccessCondition.setSelectedProcedure(item.dispenseSuccessCondition);
+//		dispenseResultItemstack.setSelectedProcedure(item.dispenseResultItemstack);
+//
+//		updateDispenseElements();

```

Comment:
Why do you keep these lines?
```suggestion
```

Reply:
I was doing some tests and then, I forgot those comments.